values.yaml

```diff
@@ -355,7 +358,7 @@ ingress:
 adhocWorker:
   # adhocWorker.env -- Redash ad-hoc worker specific envrionment variables.
   env:
-    QUEUES: "queries,celery"
+    QUEUES: "queries"
```

Comment:
Hey. I was about to make a PR for the v9 fork we've been running a while, but this one seems to incorporate all of the changes. I wish I'd known about it sooner 😄 

Just wanted to say that we've found that a few extra queues needed to be present in the ad-hoc and scheduled workers (at least in v9):
[PR](https://github.com/getredash/contrib-helm-chart/compare/master...Jawshua:9.0.0#diff-8377b3e3740a3fcd9f682e5fb55425f2fdbece1791854b9e5013e7f1a5e60e7eR361-R399)

Reply:
Thanks 👍 
I used this `docker-compose.yml` as a reference to update workers and queues.

https://github.com/getnewdash/setup/blob/5c739da578f33ab3f9fb088654de2a4de11fcf5d/data/docker-compose.yml